docs/data/data-grid/row-spanning/row-spanning.md

```diff
@@ -1,18 +1,56 @@
-# Data Grid - Row spanning 🚧
+# Data Grid - Row spanning
```

Comment:
Could you also update the table in https://deploy-preview-14124--material-ui-x.netlify.app/x/react-data-grid/getting-started/#feature-comparison?